source/routing/tests/routing_api_test_data.h

```diff
@@ -682,8 +689,15 @@ struct routing_api_data {
 
             impact->informed_entities.push_back(make_pt_obj(nt::Type_e::Route, "A:0", *b.data->pt_data, impact));
 
-            impact->messages.push_back({"no luck", "sms", "sms", "content type", default_date, default_date});
-            impact->messages.push_back({"try again", "sms", "sms", "content type", default_date, default_date});
+            channel_types.clear();
+            channel_types.insert(ChannelType::web);
+            channel_types.insert(ChannelType::sms);
+            impact->messages.push_back({"no luck", "sms", "sms", "content type", default_date, default_date, channel_types});
```

Comment:
can't you do `impact->messages.push_back({"no luck", "sms", "sms", "content type", default_date, default_date, {ChannelType::web, ChannelType::sms}});`